src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -243,6 +243,55 @@ public Task CallFunctionWithRetryAsync(string functionName, RetryOptions retryOp
         }
 
         /// <summary>
+        /// Schedules an orchestrator function named <paramref name="functionName"/> for execution.
+        /// </summary>
+        /// <param name="functionName">The name of the orchestrator function to call.</param>
+        /// <param name="parameters">The JSON-serializeable parameters to pass as input to the function.</param>
+        /// <returns>A durable task that completes when the called function completes or fails.</returns>
+        /// <exception cref="ArgumentException">
+        /// The specified function does not exist, is disabled, or is not an orchestrator function.
+        /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// The current thread is different than the thread which started the orchestrator execution.
+        /// </exception>
+        /// <exception cref="DurableTask.Core.Exceptions.TaskFailedException">
+        /// The activity function failed with an unhandled exception.
+        /// </exception>
+        public Task CallOrchestratorAsync(string functionName, params object[] parameters)
```

Comment:
Should we name it `CallSubOrchestratorAsync` to be consistent with the Durable Task Framework naming?